homeassistant/components/elkm1/climate.py

```diff
@@ -59,12 +56,12 @@
 
 async def async_setup_entry(
     hass: HomeAssistant,
-    config_entry: ConfigEntry,
+    config_entry: ElkM1ConfigEntry,
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Create the Elk-M1 thermostat platform."""
-    elk_data: ELKM1Data = hass.data[DOMAIN][config_entry.entry_id]
-    elk = elk_data.elk
+    elk_data = config_entry.runtime_data
+    elk = config_entry.runtime_data.elk
```

Comment:
To be consistent with other code change to `elk = elk_data.elk`.

Reply:
adjusted in d68e8ddf1596fd863e6d96df483d77deca557cc4